thrall/app/controllers/ThrallController.scala

```diff
@@ -217,6 +229,25 @@ class ThrallController(
     }
   }
 
+  def upsertFromProjectionSingleImage: Action[AnyContent] = withLoginRedirectAsync { implicit request =>
+    val imageId = migrateSingleImageFormReader.bindFromRequest.get.id
+
+    for {
+      maybeImage <- gridClient.getImageLoaderProjection(imageId, auth.innerServiceCall)
+    } yield { maybeImage match {
+      case Some(projectedImage) =>
+        try {
+          messageSender.publish(UpsertFromProjectionMessage(imageId, projectedImage, DateTime.now))
+          Ok(s"upsert request for $imageId submitted")
+        } catch {
+          case t: Throwable =>
+            logger.error("Submitting upsertFromProjectionMessage failed", t)
+            throw t // rethrow to get default play error message
+        }
```

Comment:
don't think this try/catch will give us much...

```suggestion
        messageSender.publish(UpsertFromProjectionMessage(imageId, projectedImage, DateTime.now))
        Ok(s"upsert request for $imageId submitted")
```